### PR TITLE
Connection endpoints

### PR DESCRIPTION
**Pull request by morrieinmaas**
Scaffolding of desired connection endpoints.

closing #48

### CODE REVIEW

**Comment by morrieinmaas** [comment]
@vineeth14 please have a look/do a code review when you find the time

**Comment by morrieinmaas** [comment]
@vineeth14 I can run the tests locally without a problem. Not sure yet why they fail in GH actions. I'll do some digging.

**Comment by morrieinmaas** [comment]
@vineeth14 ready for review 👍🤗

File: app/generic/connections.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from aries_cloudcontroller import AriesAgentControllerBase
+
+from dependencies import member_agent
+
+logger = logging.getLogger(__name__)
+
+
+router = APIRouter(prefix="/generic/connections", tags=["connections"])
+
+
```
Comment (vineeth14) [inline app/generic/connections.py:13]: Should these calls be limited to only the member_agent?

Comment (morrieinmaas) [reply app/generic/connections.py:13]: hmm I was wondering the same. Should cover most cases though imo. In the end everything you wanna connect to should be a sub wallet (except for YOMA wallet itself maybe?). My take on this is we can always create these endpoints for/extend them to whatever extra case we have. This should be the general use case though. That said if you think I'm wrong here or missing something please do say

**Comment by morrieinmaas** [comment]
@vineeth14 you can merge it and make it generic afetr #58 is merged imo

**Comment by morrieinmaas** [comment]
@vineeth14 Please have a look and merge when ready

**Comment by vineeth14** [comment]
Cool, will have a look

**Pull request merged**